target/linux/armsr/armv7/config-6.1

```diff
@@ -42,36 +62,205 @@ CONFIG_CPU_SPECTRE=y
 CONFIG_CPU_THUMB_CAPABLE=y
 CONFIG_CPU_TLB_V7=y
 CONFIG_CPU_V7=y
-CONFIG_CRYPTO_LIB_BLAKE2S_GENERIC=y
+CONFIG_CRYPTO_DEV_STM32_CRC=y
+CONFIG_CRYPTO_DEV_STM32_CRYP=y
+CONFIG_CRYPTO_DEV_STM32_HASH=y
+CONFIG_CRYPTO_SHA1=y
 CONFIG_DEBUG_LL_INCLUDE="mach/debug-macro.S"
 CONFIG_DMA_OPS=y
+CONFIG_DMA_VIRTUAL_CHANNELS=y
+CONFIG_DMA_SHARED_BUFFER=y
+CONFIG_DRM_STM=y
+CONFIG_DRM_STM_DSI=y
 CONFIG_EDAC_ATOMIC_SCRUB=y
+# CONFIG_FTM_QUADDEC is not set
 CONFIG_GENERIC_IRQ_MULTI_HANDLER=y
 CONFIG_GENERIC_VDSO_32=y
+CONFIG_GENERIC_IRQ_CHIP=y
+CONFIG_GENERIC_PHY=y
+CONFIG_GENERIC_PINCONF=y
+CONFIG_GENERIC_PINCTRL_GROUPS=y
+CONFIG_GENERIC_PINMUX_FUNCTIONS=y
+# CONFIG_KEYBOARD_ST_KEYSCAN is not set
+CONFIG_GPIO_GENERIC=y
+CONFIG_GPIO_GENERIC_PLATFORM=y
+CONFIG_GPIO_SYSCON=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_HARDEN_BRANCH_PREDICTOR=y
 CONFIG_HAVE_SMP=y
+CONFIG_HW_RANDOM=y
+CONFIG_HW_RANDOM_OPTEE=y
+CONFIG_HW_RANDOM_ST=y
+CONFIG_HW_RANDOM_STM32=y
+CONFIG_HWSPINLOCK_STM32=y
 CONFIG_HZ_FIXED=0
-CONFIG_HZ_PERIODIC=y
+CONFIG_I2C=y
+CONFIG_I2C_BOARDINFO=y
+CONFIG_I2C_SLAVE=y
+CONFIG_I2C_SMBUS=y
+CONFIG_I2C_ST=y
+CONFIG_I2C_STM32F4=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_DEMUX_PINCTRL=y
+CONFIG_I2C_MUX=y
+CONFIG_I2C_MUX_PCA954x=y
+CONFIG_I2C_MUX_PINCTRL=y
+CONFIG_I2C_STM32F7=y
+CONFIG_IIO_STM32_LPTIMER_TRIGGER=y
+# CONFIG_INTEL_QEP is not set
+# CONFIG_INTERRUPT_CNT is not set
+# CONFIG_MICROCHIP_TCB_CAPTURE is not set
+CONFIG_JFFS2_FS=y
+CONFIG_MACH_STM32MP13=y
+CONFIG_MACH_STM32MP157=y
+CONFIG_MFD_CORE=y
+# CONFIG_MFD_STM32_TIMERS is not set
+# CONFIG_MTD_ST_SPI_FSM is not set
+CONFIG_MFD_STM32_LPTIMER=y
 CONFIG_MIGHT_HAVE_CACHE_L2X0=y
+CONFIG_MMC=y
+CONFIG_MMC_ARMMMCI=y
+CONFIG_MMC_CQHCI=y
+CONFIG_MMC_RICOH_MMC=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_IO_ACCESSORS=y
+CONFIG_MMC_SDHCI_PCI=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI_ST=y
```

Comment:
I think you can drop all the SDHCI config options, AFAIK they are only used on STI.

Reply:
Remove them. Thanks